routes/cube_routes.js

```diff
@@ -2630,6 +2629,11 @@ router.get('/deck/:id', async (req, res) => {
       return res.status(404).render('misc/404', {});
     }
 
+    let draft;
+    if (deck.draft) {
+      draft = await Draft.findById(deck.draft).lean();
+    }
```

Comment:
FWIW (fine for now) this can be done by adding `.populate('draft')` to the DB query above to avoid a DB round trip.